src/index.ts

```diff
@@ -1,84 +1,77 @@
-import * as core from '@actions/core';
-import * as io from '@actions/io';
-import { BuildahCli, BuildahConfigSettings } from './buildah';
-import * as recognizer from 'language-recognizer';
-import {promises as fs} from 'fs';
-import * as path from 'path';
-import { Language } from 'language-recognizer/lib/types';
+import * as core from "@actions/core";
+import * as io from "@actions/io";
+import * as path from "path";
+import { BuildahCli, BuildahConfigSettings } from "./buildah";
 
 export async function run(): Promise<void> {
-
-    if (process.env.RUNNER_OS !== 'Linux') {
-        throw new Error('buildah, and therefore this action, only works on Linux. Please use a Linux runner.');
+    if (process.env.RUNNER_OS !== "Linux") {
+        throw new Error("buildah, and therefore this action, only works on Linux. Please use a Linux runner.");
     }
 
     // get buildah cli
-    const buildahPath = await io.which('buildah', true);
+    const buildahPath = await io.which("buildah", true);
     const cli: BuildahCli = new BuildahCli(buildahPath);
 
-    const workspace = process.env['GITHUB_WORKSPACE'];
-    let dockerFiles = getInputList('dockerfiles');
-    const newImage = `${core.getInput('image', { required: true })}:${core.getInput('tag', { required: true })}`;
-
-    const useOCI = core.getInput("oci") == "true";
+    const workspace = process.env.GITHUB_WORKSPACE || "";
```

Comment:
The action is going to fall over later if this is `""`, so that's not a reasonable default. It would be better to fail right here (throw an error) if `GITHUB_WORKSPACE` is not set. 

Or, you could use `process.cwd()` as a reasonable default - let's go with that.